portfolio/src/main/webapp/script.js

```diff
@@ -48,12 +56,52 @@ function getMessages() {
   });
 }
 
+
+
+//check whether user is logged in using async
+async function checkWhetherUserIsLoggedIn() {
+  //set visibility of form and logurl to none
+  document.getElementById('comment-form').style.display="none";
+  document.getElementById('logInOut').style.display="none";
+  
+  //get login status
+  const response = await fetch('/login');
+  const quote = await response.text();
+  console.log(quote);
+
+  //convert login information to json object
+  var obj = JSON.parse(quote);
```

Comment:
If you use `response.json()` instead of `response.text()` you can skip this step. Or do you prefer keeping it as two different steps like this?